core/migrations/populate_officeblock_location.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+
+def update_officeblocks_location(apps, schema_editor):
+    office_block = apps.get_model('core', 'OfficeBlock')
+    andela_centres = apps.get_model('core', 'AndelaCentre')
+    block_location = None
+    for office_block in office_block.objects.all():
+        # find kenya centre
+        centre = andela_centres.objects.filter(country="Kenya")[0]
```

Comment:
I fear `andela_centres.objects.filter(country="Kenya")[0]` might fail with an **IndexError** if no center has a **country** named **kenya**. We could use `andela_centres.objects.filter(country="Kenya").exists()` instead.